### PR TITLE
Added Safety Alerts Channel

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
[contributing]: https://jda.wiki/contributing/contributing/

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

Added getSafetyAlertsChannel allowing you to get the channel to which Discord sends alerts to.

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/api/managers/GuildManager.java [hunk 1]
```diff
@@ -78,6 +78,10 @@ public interface GuildManager extends Manager<GuildManager>
     long BOOST_PROGRESS_BAR_ENABLED = 1 << 14;
     /** Used to add or remove modifiable features (such as {@code "INVITES_DISABLED"}) */
     long FEATURES = 1 << 15;
+    /**
+     * Used to rest the safety alerts channel field.
+     */
```
Comment (freya022) [inline src/main/java/net/dv8tion/jda/api/managers/GuildManager.java:83]:
```suggestion
    /** Used to rest the safety alerts channel field */
```

File: src/main/java/net/dv8tion/jda/api/managers/GuildManager.java [hunk 2]
```diff
@@ -259,6 +263,22 @@ public interface GuildManager extends Manager<GuildManager>
     @CheckReturnValue
     GuildManager setCommunityUpdatesChannel(@Nullable TextChannel communityUpdatesChannel);
 
+    /**
+     * Sets the safety alerts {@link net.dv8tion.jda.api.entities.channel.concrete.TextChannel TextChannel} of this {@link net.dv8tion.jda.api.entities.Guild Guild}.
```
Comment (freya022) [inline src/main/java/net/dv8tion/jda/api/managers/GuildManager.java:267]:
```suggestion
     * Sets the safety alerts {@link TextChannel} of this {@link Guild}.
```

File: src/main/java/net/dv8tion/jda/api/managers/GuildManager.java [hunk 3]
```diff
@@ -259,6 +263,22 @@ public interface GuildManager extends Manager<GuildManager>
...
+     *
+     * @param  safetyAlertsChannel
+     *         The new safety alerts channel for this {@link net.dv8tion.jda.api.entities.Guild Guild}
```
Comment (freya022) [inline src/main/java/net/dv8tion/jda/api/managers/GuildManager.java:270]:
```suggestion
     *         The new safety alerts channel for this {@link Guild}
```

File: src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java [hunk 4]
```diff
@@ -632,6 +633,12 @@ public TextChannel getCommunityUpdatesChannel()
         return communityUpdatesChannel;
     }
 
+    @Override
```
Comment (freya022) [inline src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java:636]:
```suggestion
    @Nullable
    @Override
```

**Review by freya022 — CHANGES_REQUESTED** [review]
You don't need the put the fully qualified name in the javadocs if the type is already imported

**Review by freya022 — APPROVED** [review]
Looks fine, I haven't compared to similar classes though

**Review by DV8FromTheWorld — APPROVED** [review]
LGTM!

**Pull request merged**